src/mainwindow.cpp

```diff
@@ -1199,13 +1199,15 @@ void MainWindow::on_actionShowExifData_triggered(bool checked) {
 }
 
 void MainWindow::setShowThumbnails(bool show) {
+  Settings& settings = static_cast<Application*>(qApp)->settings();
```

Comment:
For now, `settings` is used only in one place inside this function. Logically, it's better to put it before the line, where it's used (although there's no practical change).

Reply:
It will be used more in the next PR (to choose the dock position).  Would you still want it moved back and forth between the PRs?

Reply:
I went ahead and moved it.